src/foam/mlang/mlang.js

```diff
@@ -1862,7 +1864,7 @@ foam.CLASS({
       return typeof x === 'number' ? '' + x :
         typeof x === 'string' ? '"' + x + '"' :
         Array.isArray(x) ? '[' + x.map(this.toString_.bind(this)).join(', ') + ']' :
-        x && (x).toString();
+        x && x.toString && x.toString();
```

Comment:
BTW, you can now do this in JS:
x && x.toString?.()

Reply:
updated